docs/developer/community/faq.mdx

```diff
@@ -0,0 +1,150 @@
+
+- `API_URI` (required): URI of the GraphQL API endpoint of a running instance of Saleor. If you are running Saleor locally with the default settings, set `API_URI` to: `http://localhost:8000/graphql/`.
+
+- `APP_MOUNT_URI`: URI at which the Dashboard app will be mounted. E.g. if you set `APP_MOUNT_URI` to `/dashboard/``, your app will be mounted at `http://localhost:9000/dashboard/`.
+
+- `STATIC_URL`: URL where the static files are located. E.g. if you use S3 bucket, you should set it to the bucket's URL. By default, Saleor assumes you serve static files from the root of your site at `http://localhost:9000/`.
+
+You can find the instructions on how to install and run the dashboard here:
+
+https://github.com/mirumee/saleor-dashboard#getting-started
+
+If you still cannot log in to the dashboard, please contact us on Spectrum.
+
+## How to make Saleor work in production with Ubuntu 19.10/Nginx/Gunicorn server?
+
+You will need to perform the following actions:
+
+- Build the storefront and the dashboard.
+
+- Upload both to separate folders on the server.
+
+- Create separate location entries in your Nginx config for the backend, the dashboard, and the storefront.
+
+    ```
+    location /graphql/ {
+        proxy_pass http://localhost:8000;
+    }
+
+    location / {
+        root /path/to/your/storefront;
+    }
+
+    location /dashboard/ {
+        root /path/to/your/dashboard;
+    }
+    ```
+
+If you still have any issues, please contact us on Spectrum.
+
+## I receive an error 405. What does it mean?
+
+- Please check if you have set the environment variables API_URI
+
+- Please check if the app is actually connected to the API
+
+- You can investigate the connection in the Network tab in your browser’s dev tools. Please look for GraphQL requests.
+
+If you still have any issues, please [contact us on Spectrum](https://spectrum.chat/saleor) or [Gitter](https://gitter.im/mirumee/saleor).
+
+## How can I run Saleor locally?
+
+We’ve recently introduced a new repository: [saleor-platform](https://github.com/mirumee/saleor-platform). You can run all Saleor components (GraphQL API, Storefront, Dashboard) from one place by using Docker. Please, bear in mind that Saleor-Platform repository has only been designed for local development. If you still have any issues, please contact us on Gitter or Spectrum.
+
+## How can I add a custom payment gateway to Saleor?
+
+See [Extending Saleor](developer/extending/plugins.mdx).
+
+## How to set up the environment variables?
+
+See [Configuration](developer/running-saleor/configuration.mdx#setting-environment-variables).
+
+## Why is Razorpay not available in the storefront?
+
+Razorpay is currently only supported for implementation on the API. You need to separately prepare your storefront for using Razorpay.
+
+## After deploying Saleor on Heroku, the platform is not working. What should I do?
+
+You need to configure all three apps (GraphQL API, Dashboard, Storefront) in Nginx:
```

Comment:
Is the part with Nginx true for Heroku to deploy these three components?

Reply:
No idea, I'm just the messenger here, content comes from @karolkielecki.

Reply:
As far as I remember it was written by one of our community members. According to another community member, it worked like that. I don't have a link for the thread. If we are not sure for 100% then we don't need to publish it right now. But we should investigate it further - many people have issues with deployment on Heroku.

Reply:
Let's ask our Cloud team ie. Miłosz. They should be able to confirm it for us. Then and only then we can publish it. It's an important question so we should provide an answer but we need to be certain that it works and that we're publishing valuable stuff only in general.

Reply:
In that case, I suggest we drop this from the FAQ and add it to the deployment docs.